### PR TITLE
Feature/stylize and add new functions form user #3

### PR DESCRIPTION
**Pull request by jossee0219**
Add new features form User #3

### CODE REVIEW

File: src/FormUser.js [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+    constructor(){
+        super();
+        this.state = { users: [], nameText: '', lastNameText: '', emailText: '', phoneNumberText: '', countryText: '', addressText: '', shown: true};
+        this._handleChangeFirstName = this._handleChangeFirstName.bind(this);
+        this._handleChangeLastName = this._handleChangeLastName.bind(this);
+        this._handleChangeCountry = this._handleChangeCountry.bind(this);
+        this._handleChangeAddress = this._handleChangeAddress.bind(this);
+        this._handleChangeEmail = this._handleChangeEmail.bind(this);
+        this._handleChangePhoneNumber = this._handleChangePhoneNumber.bind(this);
+        this._handleClick = this._handleClick.bind(this);
```
Comment (serudda) [inline src/FormUser.js]: Rename to autoexplicity name, please

File: src/FormUser.js [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+        this._handleClickDelete = this._handleClickDelete.bind(this);
+        this._handleClickEdit = this._handleClickEdit.bind(this);
+        this._handleClickSaveEdit = this._handleClickSaveEdit.bind(this);
+    }
+
+    /*toggle() {
```
Comment (serudda) [inline src/FormUser.js]: Remove Comments

File: src/FormUser.js [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+
+    _handleClick(){
+        let newUser = {
+            id : Date.now(),
+            firstname : this.state.nameText,
+            lastname: this.state.lastNameText,
+            country: this.state.countryText,
+            address: this.state.addressText,
+            email: this.state.emailText,
+            phonenumber: this.state.phoneNumberText
```
Comment (serudda) [inline src/FormUser.js]: Rename: phonenumber property to 'phoneNumber' (camelCase)

File: src/FormUser.js [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+            if (user.id === id){
+              return user
+            } 
+        }, this);
+        
+        let index = this.state.users.findIndex(user => 
+            user.id === id
+        ); 
+
+        console.log(index)
```
Comment (serudda) [inline src/FormUser.js]: Remove console.log

File: src/FormUser.js [hunk 5]
```diff
@@ -0,0 +1,262 @@
...
+            lastNameText: user[index].lastname,
+            countryText: user[index].country,
+            addressText: user[index].address, 
+            emailText: user[index].email,
+            phoneNumberText: user[index].phonenumber,
+            shown: !this.state.shown
+        })
+
+        return(
+           <button> Take karete take</button>
```
Comment (serudda) [inline src/FormUser.js]: Remove or change?

File: src/FormUser.js [hunk 6]
```diff
@@ -0,0 +1,262 @@
...
+            if (user.id === id){
+                return user
+            } 
+        }, this);
+        
+        let index = this.state.users.findIndex(user => 
+            user.id === id
+        ); 
+
+        console.log(index)
```
Comment (serudda) [inline src/FormUser.js]: remove

File: src/FormUser.js [hunk 7]
```diff
@@ -0,0 +1,262 @@
...
+
+        this.setState({
+            users: users.filter((user,i) =>
+                index !== i
+            )
+        })
+      }    
+
+    render(){
+        var shown = {
```
Comment (serudda) [inline src/FormUser.js]: Change to 'let' or 'const'

**Review by serudda — CHANGES_REQUESTED** [review]

File: src/FormUser.js (hunk 1 above)
Comment (jossee0219) [reply src/FormUser.js]: Done

File: src/FormUser.js (hunk 2 above)
Comment (jossee0219) [reply src/FormUser.js]: Done

File: src/FormUser.js (hunk 3 above)
Comment (jossee0219) [reply src/FormUser.js]: Done

File: src/FormUser.js (hunk 4 above)
Comment (jossee0219) [reply src/FormUser.js]: Done

File: src/FormUser.js (hunk 5 above)
Comment (jossee0219) [reply src/FormUser.js]: Done

File: src/FormUser.js (hunk 6 above)
Comment (jossee0219) [reply src/FormUser.js]: Done

File: src/FormUser.js (hunk 7 above)
Comment (jossee0219) [reply src/FormUser.js]: Done

**Review by serudda — APPROVED** [review]

**Pull request merged**